cate/ds/esa_cci_odp.py

```diff
@@ -794,6 +794,12 @@ async def _create_data_source(self, json_dict: dict, datasource_id: str):
             if drs_id == 'esacci.AEROSOL.satellite-orbit-frequency.L2P.AER_PRODUCTS.ATSR-2.ERS-2.SU.4-21.r1' and \
                     datasource_id == '59f3a38819e140b49ffe46f32176709e':
                 continue
+            if drs_id == 'esacci.AEROSOL.day.L3C.AER_PRODUCTS.ATSR-2.Envisat.ATSR2.v2-6.r1' and \
+                    datasource_id == 'c183044b88734442b6d37f5c4f6b0092':
+                continue
+            if drs_id == 'esacci.AEROSOL.satellite-orbit-frequency.L2P.AER_PRODUCTS.AATSR.Envisat.AATSR-ENVISAT-ENS.v2-6.r1' and \
+                    datasource_id == '4afb736dc395442aa9b327c11f0d704b':
+                continue
```

Comment:
Leave a comment explaining what's going on here! Or refactor out method so you can write the following:

```suggestion
            if self.is_dataset_dropped(drs_id, datasource_id):
                continue
```

Reply:
Then you can document `is_dataset_dropped` and explain why it is needed.